add reuseble components

MykhailoKutsyi opened the pull request:
Hello, there

Connected  owner profile page.

App wrapped in a container with breaking points.

Added reusable component:
- Button, styled with styled-components.

For now, this page :

![image](https://user-images.githubusercontent.com/91030243/201734227-b439749a-8877-4f2b-8e11-1614226d9bcb.png)

MykhailoKutsyi committed:
add reuseble components

lyudad commented on libs/components/src/lib/container/container.tsx:
```diff
@@ -0,0 +1,12 @@
+import { ContainerWrapper } from './styles';
+
+/* eslint-disable-next-line */
```
why so many eslint disable, we create the rules to follow them

MykhailoKutsyi replied:
In this case - dont need there, I have props
it came from created component, just missed out it

MykhailoKutsyi committed:
add JobPostPage, add JobPostForm

MykhailoKutsyi commented:
Hello
added JobPostPage, added JobPostForm 

For now, have form with validation. Connect Selects to Form with Controller from react-hook-form
Styles would be updated.

And also make reuseble component Icon (return svg). Which help more easy include nice svg to project.

MykhailoKutsyi commented:
When data entered and pressed submit, object collected all data and shown at console, for now, temporary)
Screen 
![image](https://user-images.githubusercontent.com/91030243/202217896-679ddc01-fa6d-4643-bf93-cd238a2bb240.png)

lyudad commented on libs/components/src/lib/JobPostForm/index.tsx:
```diff
@@ -0,0 +1,124 @@
+
+import Icon from '../assets/Icon';
+import { StyledButton } from '../button/styles';
+import { multipleSelectOption, options } from '../constants/options';
```
absolute path

lyudad commented on libs/components/src/lib/JobPostForm/styles.ts:
```diff
@@ -0,0 +1,66 @@
+  ::placeholder {
+    font-size: 18px;
+    line-height: 32px;
+    color: #2d2d2d;
```
<img width="744" alt="Снимок экрана 2022-11-15 в 20 29 24" src="https://user-images.githubusercontent.com/25103090/202297957-313b41cb-4a02-4965-819a-bc71d65f0f99.png">

MykhailoKutsyi committed:
update path

MykhailoKutsyi committed:
Merge branch 'dev' of https://github.com/ZenBit-Tech/coffee_lovers_frontend into add/ownerPage
